src/Identity/src/Core/Microsoft.Extensions.Identity.Core.csproj

```diff
@@ -1,8 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Random question, why is this line showing up as a diff?

Reply:
The UTF8 BOM probably changed